src/components/decimal/decimal.test.tsx

```diff
@@ -1116,7 +1116,7 @@ describe("when the component is uncontrolled", () => {
   it("renders a negative value with correct formatting when the `it` locale is set", () => {
     render(<Decimal defaultValue="-1234.56" locale="it" />);
 
-    expect(screen.getByRole("textbox")).toHaveValue("-1.234,56");
+    expect(screen.getByRole("textbox")).toHaveValue("-1234,56");
```

Comment:
comment (non-blocking): it's probably safer to use `Intl.NumberFormat` here so that any future updates to ICU don't break this test but it's not a critical issue as we plan to deprecate uncontrolled support and potentially `Decimal` at some point anyway

Reply:
Actually I think you'll have to do this to prevent it failing in node 18